xml/MAIN.susestorage.xml

```diff
@@ -25,6 +25,7 @@
    <dm:bugtracker>
     <dm:url>https://github.com/SUSE/doc-ses/issues/new/choose</dm:url>
```

Comment:
You need to remove `/choose` from the URL because that is still incompatible with us submitting our own issue body.